app/views/admin/shared/_admin_menu.html.erb

```diff
@@ -5,5 +5,6 @@
     <li><%= link_to "Dashboard", admin_path %></li>
     <li><%= link_to "Manage Items", items_path %></li>
     <li><%= link_to "Manage Categories", categories_path %></li>
+    <li><%= link_to "Manage Users", admin_users_path %></li>
```

Comment:
Maybe change this to "View Customers" or "View Users"? (since there is no managing going on).